.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,25 @@
+name: algolia-scraper
+on:
+  push: # only run on push events
+    branches:
+      - main
+  check_suite: # only run check suite events
+    types: [completed]
+  workflow_dispatch: # only run on workflow dispatch events
+jobs:
+  scrape:
+    runs-on: ubuntu-latest # specifies the OS image to use for the job
+    steps: # specifies the commands to run
+      - name: check out code 🛎
+        uses: actions/checkout@v2
+      - name: scrape the site 🧽
+        env: #- specifies the environment variables
+          APPLICATION_ID: ${{ secrets.APPLICATION_ID }} #generated when you create a project on algolia website after you sign up
+          WRITE_API_KEY: ${{ secrets.API_KEY }} #API key be sure to have the right permissions
+          SITE_URL: ${{ secrets.SITE_URL }} #site url
+          INDEX_NAME: ${{ secrets.INDEX_NAME }} #docusaurus or anything if you want ot change it
+          # run: create .env file and run bash script scrape.sh
+        run: |
+          touch .env  
```

Comment:
What's the purpose of having an empty `.env`?

Reply:
the bash script needs to use those variables